src/TestFramework/PhpUnit/Config/Builder/MutationConfigBuilder.php

```diff
@@ -54,29 +55,31 @@
 class MutationConfigBuilder extends ConfigBuilder
 {
     use SafeQuery;
-    private $tempDirectory;
+
+    private $tmpDir;
     private $projectDir;
+    private $originalXmlConfigContent;
     private $xmlConfigurationHelper;
-    private $dom;
     private $jUnitTestCaseSorter;
 
+    /**
+     * @var DOMDocument|null
+     */
+    private $dom;
+
     public function __construct(
-        string $tempDirectory,
+        string $tmpDir,
         string $originalXmlConfigContent,
         XmlConfigurationHelper $xmlConfigurationHelper,
         string $projectDir,
         JUnitTestCaseSorter $jUnitTestCaseSorter
     ) {
-        $this->tempDirectory = $tempDirectory;
+        $this->tmpDir = $tmpDir;
         $this->projectDir = $projectDir;
 
+        $this->originalXmlConfigContent = $originalXmlConfigContent;
         $this->xmlConfigurationHelper = $xmlConfigurationHelper;
         $this->jUnitTestCaseSorter = $jUnitTestCaseSorter;
-
-        $this->dom = new DOMDocument();
```

Comment:
There is no reason to do this in the constructor which adds to the cost of instantiating the builder, instead I moved it in a memoized getter to evaluate this code lazily